src/modules/vk.js

```diff
@@ -1,91 +1,93 @@
 // Vkontakte (vk.com)
-(function(hello) {
-
-	hello.init({
-
-		vk: {
-			name: 'Vk',
-
-			// See https://vk.com/dev/oauth_dialog
-			oauth: {
-				version: 2,
-				auth: 'https://oauth.vk.com/authorize',
-				grant: 'https://oauth.vk.com/access_token'
-			},
-
-			// Authorization scopes
-			// See https://vk.com/dev/permissions
-			scope: {
-				email: 'email',
-				friends: 'friends',
-				photos: 'photos',
-				videos: 'video',
-				share: 'share',
-				offline_access: 'offline'
-			},
-
-			// Refresh the access_token
-			refresh: true,
-
-			login: function(p) {
-				p.qs.display = window.navigator &&
-					window.navigator.userAgent &&
-					/ipad|phone|phone|android/.test(window.navigator.userAgent.toLowerCase()) ? 'mobile' : 'popup';
-			},
-
-			// API Base URL
-			base: 'https://api.vk.com/method/',
-
-			// Map GET requests
-			get: {
-				me: function(p, callback) {
-					p.query.fields = 'id,first_name,last_name,photo_max';
-					callback('users.get');
-				}
-			},
-
-			wrap: {
-				me: function(res, headers, req) {
-					formatError(res);
-					return formatUser(res, req);
-				}
-			},
-
-			// No XHR
-			xhr: false,
-
-			// All requests should be JSONP as of missing CORS headers in https://api.vk.com/method/*
-			jsonp: true,
-
-			// No form
-			form: false
+if (typeof window !== 'undefined') {
```

Comment:
Not the whole library relies on this.

Reply:
Would it be better to take this one out then? 
I think I saw the reference to ```window.navigator.userAgent``` on line 32 and assumed it would throw an error if window wasn't present.